docs/script-api.md

```diff
@@ -893,6 +905,14 @@ end, ON.LEVEL)
 - [`TITLE_TORCHFLAME_SMOKE`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PARTICLEEMITTER.TITLE_TORCHFLAME_SMOKE) 1
 - ...check particle_emitters.txt output by Overlunky...
 - [`MINIGAME_BROKENASTEROID_SMOKE`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PARTICLEEMITTER.MINIGAME_BROKENASTEROID_SMOKE) 219
+### DROPCHANCE
+- [`BONEBLOCK_SKELETONKEY`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=DROPCHANCE.BONEBLOCK_SKELETONKEY) 0
+- ...see_drops.hpp_for_a_list_of_possible_dropchances...
```

Comment:
Here it looks like you didn't need `_` for the docs to not break.

Reply:
Do you mean inside the string? Like for_a_list etc? 
All the other comments like this, (including your sound_params :) ) do the same though. I'll check and see if it works without.

Reply:
This is another ultra-l33t-hax0r thing in the docs generator. It will replace double underscores in this context with spaces and we can't use spaces directly because some other part strips spaces but we also can't use single underscores because we use those for file names.